internal/pkg/service/buffer/store/schema/token.go

```diff
@@ -0,0 +1,33 @@
+package schema
+
+import (
+	"strconv"
+
+	storeKey "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
+	. "github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop"
+)
+
+type tokens = PrefixT[model.Token]
+
+type Tokens struct {
+	tokens
+}
+
+type TokensInExport struct {
+	tokens
+}
+
+func (v SecretsRoot) Tokens() Tokens {
+	return Tokens{tokens: NewTypedPrefix[model.Token](
+		v.prefix.Add("export/token"),
+		v.schema.serialization,
+	)}
+}
+
+func (v Tokens) InExport(k storeKey.ExportKey) KeyT[model.Token] {
+	return v.tokens.
+		Add(strconv.Itoa(k.ProjectID)).
+		Add(k.ReceiverID).
+		Key(k.ExportID)
```

Comment:
Tu prosim sprav panic, ak su tie hodnoty prazdne, vid 
https://github.com/keboola/keboola-as-code/blob/9915ed6917b31b07b83f7fab0cd7c32dbf0f6295/internal/pkg/service/buffer/store/schema/record.go#L43-L63

To ma byt posledna ochrana proti tomu, aby si omylom pouzil nejaky iny token.
Ak by bol ExportKey uplne prazdny a neprideme na to.
Tak to ulozis ako `secrets/export/token////` a tak aj nacitas, co by bolo nebezpecne.